cves/CVE-2013-1664.yml

```diff
@@ -1,296 +1,314 @@
+  upvotes to each vulnerability you see. Your peers will tell you how
+  interesting they think this vulnerability is, and you'll add that to the
+  upvotes score on your branch.
+upvotes:
+unit_tested:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  code: true
+  code_answer: true
+  fix: true
+  fix_answer: true
+discovered:
+  question: |
+    How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please explain where you looked.
+  answer: |
+    Was found by a Yahoo! employee when DOS was plaguing Nova keystone servers
+  automated: false
+  contest: false
+  developer: true
+autodiscoverable:
+  instructions: |
+    Is it plausible that a fully automated tool could have discovered
+    this? These are tools that require little knowledge of the domain,
+     e.g. automatic static analysis, compiler warnings, fuzzers.
+
+    Examples for true answers: SQL injection, XSS, buffer overflow
+
+    Examples for false: RFC violations, permissions issues, anything
+    that requires the tool to be "aware" of the project's
+    domain-specific requirements.
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+  answer_note: |
+    This could be analysed by Fuzzers by finding any http timeout errors, or by
+    monitoring the processes running.
+  answer: true
+specification:
+  instructions: |
+    Is there mention of a violation of a specification? For example,
+    an RFC specification, a protocol specification, or a requirements
+    specification.
+
+    Be sure to check all artifacts for this: bug report, security
+    advisory, commit message, etc.
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+  answer_note: |
+    There was a violation in XML standards that are used.

```

Comment:
Maybe a bit more specific on what the standards are about?

Reply:
Is it really a violation of XML standards? If the code wasn't proper XML, it wouldn't run would it?